docs/general/administration/hardware-acceleration/apple.md

```diff
@@ -0,0 +1,94 @@
+---
+uid: admin-hardware-acceleration-apple
+title: Apple Mac
+---
+
+# HWA Tutorial On Mac
+
+This tutorial guides you on setting up full video hardware acceleration on Apple Macs via VideoToolbox.
+
+## Acceleration Methods
+
+[VideoToolbox](https://developer.apple.com/documentation/videotoolbox) is the only available method on macOS.
+
+To achieve full acceleration, [Metal](https://developer.apple.com/metal/) and OpenCL is also required.
+
+## Tone-mapping Methods
+
+Hardware accelerated HDR/DV to SDR tone-mapping is supported on most Macs from 2017 and later, with the exception of the MacBook Air (13-inch, 2017).
+
+There are two different methods that can be used. Pros and cons are listed below:
+
+1. **OpenCL**
+
+   - Pros - Supports Dolby Vision P5, detailed fine-tuning options.
+
+   - Cons - Slower, requires more powerful GPUs.
+
+2. **VideoToolbox Native**
+
+   - Pros: Lower power consumption, less dependency on GPU performance, good visual quality without manual fine-tuning.
+
+   - Cons: Lack of tuning options, potential performance issues on Intel Macs.
+
+:::note
+
+When both methods are enabled, VideoToolbox Native will be used for most videos, and OpenCL will only be used as a fallback for Dolby Vision Profile 5 videos.
+
+:::
+
+## Select System Hardware
+
+Hardware accelerated transcoding is supported on all Macs that support [VideoToolbox](https://developer.apple.com/documentation/videotoolbox). This includes most 2011 and later Macs.<sup>*</sup>
+
+Full acceleration is available on most Macs from 2017 and later, with the exception of the MacBook Air (13-inch, 2017).
+
+To have native Apple Silicon support, Jellyfin server 10.9.0+ and the latest `jellyfin-ffmpeg6` is required.
+
+<sup>*</sup> On very old Macs, VideoToolbox may fallback to software decoding/encoding. Macs from 2017 and later is strongly recommend.
+
+:::note
+
+While hardware acceleration via VideoToolbox might work on older series Macs, it is not officially supported.
+
+:::
+
+### Transcode H.264
+
+AVC / H.264 8-bit is still widely used due to its excellent compatibility. All Intel GPUs that support QSV can decode and encode it.
```

Comment:
There exist macs out there that don't have Intel Integrated graphics (eg. Mac Pro, iMac Pro). What about these macs?